qmetaobject/src/listmodel.rs

```diff
@@ -259,6 +259,9 @@ impl<T: SimpleListItem> SimpleListModel<T> {
         self.values = data;
         (self as &mut dyn QAbstractListModel).end_reset_model();
     }
+    pub fn iter(&self) -> std::slice::Iter<'_, T> {
```

Comment:
should it maybe return `impl Iterator<...>`  ?

Reply:
+1. Can change + add docs and tests tomorrow if you like :)